packages/webapp/pages/posts/[id].tsx

```diff
@@ -127,7 +126,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
         position={position}
         post={post}
         isFallback={isFallback}
-        isLoading={isLoading}
+        isLoading={isLoading || !isFetched}
```

Comment:
I think this is the right one

```suggestion
        isLoading={!isFetched}
```